src/tokens/PositionItem.tsx

```diff
@@ -36,50 +45,57 @@ export const PositionItem = ({
       description: position.displayProps.description,
       balanceUsd: balanceUsd.toNumber(),
     })
+    earnPositionIds.includes(position.positionId)
+      ? navigate(Screens.EarnPoolInfoScreen, { pool: position as EarnPosition })
+      : position.displayProps.manageUrl
+        ? Platform.OS === 'android'
+          ? navigate(Screens.WebViewScreen, {
+              uri: position.displayProps.manageUrl,
+            })
+          : dispatch(openUrl(position.displayProps.manageUrl, true))
+        : null
```

Comment:
nit: style to avoid nested ternary. 

```suggestion
    const uri = position.displayProps.manageUrl
    if (earnPositionIds.includes(position.positionId)) {
      navigate(Screens.EarnPoolInfoScreen, { pool: position as EarnPosition })
    } else if (uri) {
      Platform.OS === 'android'
        ? navigate(Screens.WebViewScreen, { uri })
        : dispatch(openUrl(uri, true))
    }
```